### PR TITLE
[Fix] pyfancy installation without pre-installed user dependencies

### PR DESCRIPTION
**Pull request by Kristinita**
### 1. Summary

I rewrite configuration files, if user don't have local pandoc and pypandoc.

If you accept this pull-request, please, make [**these actions**](http://kristinita.ru/IT-articles/How-to-publish-your-package-to-PyPI-2018#pypi).

### 2. Behavior before pull request

I try install pyfancy use [**virtual environment**](http://kristinita.ru/IT-articles/How-to-publish-your-package-to-PyPI-2018#virtualenv):

```shell
C:\Users\SashaChernykh>mkvirtualenv sashapyfancy
Using base prefix 'c:\\python36'
New python executable in C:\Users\SashaChernykh\Envs\sashapyfancy\Scripts\python.exe
Installing setuptools, pip, wheel…done.

(sashapyfancy) C:\Users\SashaChernykh>pip install pyfancy
Collecting pyfancy
  Using cached pyfancy-2.4.4.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "C:\Users\SASHAC~1\AppData\Local\Temp\pip-build-k7_ido_t\pyfancy\setup.py", line 14, in <module>
        import pypandoc
    ModuleNotFoundError: No module named 'pypandoc'

    During handling of the above exception, another exception occurred:

    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\SASHAC~1\AppData\Local\Temp\pip-build-k7_ido_t\pyfancy\setup.py", line 17, in <module>
        long_description = open('README.md').read()
    FileNotFoundError: [Errno 2] No such file or directory: 'README.md'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in C:\Users\SASHAC~1\AppData\Local\Temp\pip-build-k7_ido_t\pyfancy
```

### 3. Behavior after pull request

```shell
C:\Users\SashaChernykh>mkvirtualenv sashafixedpyfancy
Using base prefix 'c:\\python36'
New python executable in C:\Users\SashaChernykh\Envs\sashafixedpyfancy\Scripts\python.exe
Installing setuptools, pip, wheel…done.

(sashafixedpyfancy) C:\Users\SashaChernykh>pip install --extra-index-url https://test.pypi.org/simple/ pyfancy
Collecting pyfancy
  Using cached https://test-files.pythonhosted.org/packages/fe/22/83c8cd5b4dec6c39cbd8e302d670055f8dc0556f1c99a552736bb0685c2c/pyfancy-2.4.5-py2.py3-none-any.whl
Installing collected packages: pyfancy
Successfully installed pyfancy-2.4.5
```

Also I publish pyfancy to TestPyPI, please, visit:

+ **<https://testpypi.python.org/pypi/pyfancy>**,
+ **<https://test.pypi.org/project/pyfancy/>**.

See [**more information**](http://kristinita.ru/IT-articles/How-to-publish-your-package-to-PyPI-2018#testpypi) about TestPyPI.

### 4. Argumentation

[**`README.md` — is a not good solution**](http://kristinita.ru/IT-articles/How-to-publish-your-package-to-PyPI-2018#readme) for Python packages. I convert your `README.md` to `README.rst` use [**pandoc**](http://pandoc.org).

### 5. Testing environment

See [**here**](http://kristinita.ru/IT-articles/How-to-publish-your-package-to-PyPI-2018#testing-environment).

Thanks.

### CODE REVIEW

**Comment by CosmicWebServices** [comment]
Does github read .rst?

**Comment by CosmicWebServices** [comment]
Also when I uploaded to pip the first time I needed to put the stuff from setup.cfg to setup.py was I doing it wrong or should we put the setup.cfg to setup.py

File: setup.cfg [hunk 1]
```diff
@@ -3,11 +3,12 @@
...
 description = "Simple terminal formatting in Python"
+long_description = file: README.rst
 # long_description = file: README.MD
 # Versions should comply with PEP440.  For a discussion on single-sourcing
 # the version across setup.py and the project code, see
 # https://packaging.python.org/en/latest/single_source_version.html
-version = 2.4.4
+version = 2.4.5
 # Author details
 author = CosmicWebServices
```
Comment (CosmicWebServices) [inline setup.cfg:13]: New author is pyfancy

File: setup.cfg [hunk 2]
```diff
@@ -3,11 +3,12 @@
...
 author_email = andrewrodebaugh@cosmicsearch.org
```
Comment (CosmicWebServices) [inline setup.cfg:14]: New email is aquahydration@gmail.com

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,199 @@
+|Build Status| |FOSSA Status Shield Badge|
```
Comment (CosmicWebServices) [inline README.rst:1]: Does RST files view on github?

**Review by CosmicWebServices — CHANGES_REQUESTED** [review]
I have some changes I need to make.

File: README.rst [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+
+`jonathan50`_
+
+`rogersouza`_
+
+`vutondesign`_
+
+`cruxicheiros`_
+
+`hiccup01`_
```
Comment (CosmicWebServices) [inline README.rst:182]: Also put your username if you want...

File: README.rst (hunk 3 above)
Comment (Kristinita) [reply README.rst:1]: @CosmicWebServices , yes, for example, see [**my fork**](https://github.com/Kristinita/Pyfancy-2/tree/SashaGoddess).

Thanks.

File: setup.cfg (hunk 2 above)
Comment (Kristinita) [reply setup.cfg:14]: @CosmicWebServices , you is Pyfancy author and you can change values for `author` and `author_email` parameters as you want.

Thanks.

File: README.rst (hunk 4 above)
Comment (Kristinita) [reply README.rst:182]: @CosmicWebServices , you can add me, if you want 😆 .

Thanks.

**Comment by Kristinita** [comment]
> Also when I uploaded to pip the first time I needed to put the stuff from setup.cfg to setup.py was I doing it wrong or should we put the setup.cfg to setup.py.

We can add parameters to `setup.cfg` or `setup.py`, but I think, that `setup.cfg` is more convenient. [**See also**](http://kristinita.ru/IT-articles/How-to-publish-your-package-to-PyPI-2018#setupcfg).

Thanks.

**Review by CosmicWebServices — APPROVED** [review]

**Comment by Kristinita** [comment]
@CosmicWebServices ,

>If you accept this pull-request, please, make [**these**](http://kristinita.ru/IT-articles/How-to-publish-your-package-to-PyPI-2018#pypi) actions.

Thanks.

**Comment by CosmicWebServices** [comment]
Yep gonna do this soon... I just need a real computer first (on my chromebook)

**Comment by Kristinita** [comment]
@CosmicWebServices , we wait, when you push new version to PyPI.

Thanks.

**Comment by CosmicWebServices** [comment]
Oh, I am so sorry I totally forgot... I just need to do homework then I will do it

**Comment by Kristinita** [comment]
@CosmicWebServices, ping.

If you can't update pyfancy on PyPI, please, make these steps:

Login on **<https://pypi.python.org>** → visit **<https://pypi.python.org/pypi/pyfancy/>** → `roles`:

Roles

+ `User Name` — `Sasha_Chernykh`,
+ `Role to Add` — `Maintainer`.

`Add Role`

Add Role

If you add me to maintainers, I can publish new pyfancy version to PyPI.

Thanks.

**Comment by CosmicWebServices** [comment]
Done sorry I have had 0 time to do this

**Comment by Kristinita** [comment]
@CosmicWebServices , [**done**](https://pypi.python.org/pypi/pyfancy).

Now any user can install pyfancy, if print in a terminal:

    pip install pyfancy

Thanks.

**Comment by CosmicWebServices** [comment]
No thank you :D

**Pull request merged**